delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/AclsRoutes.scala

```diff
@@ -202,6 +190,13 @@ class AclsRoutes(identities: Identities, acls: Acls)(implicit
     }
   }
 
+  private def notFoundToNone(result: IO[AclRejection, AclResource]): IO[AclRejection, Option[AclResource]] =
```

Comment:
Why do we need this notFound to None ?

Reply:
Acls is an 'special' endpoint which always return a listing, even on `AclNotFound`.